resources/charts/bitcoincore/values.yaml

```diff
@@ -118,6 +118,9 @@ regtestConfig: |
   rpcport=18443
   rpcallowip=0.0.0.0/0
   rpcbind=0.0.0.0
+  rpcauth=forkobserver:ef3d61b7ffecd81ffaaa7ae43091543d$a10bc25b5ca8910ebbee9b1538d38242b09d0a165e220f9a47535eccc4c089a4
```

Comment:
Probably don't want to set this as a default because I don't think we will be able to override it with a user supplied config, which means we will need a code change every time we want to change the password (ew). I'd recommend moving this rpcauth code black to the `node-defaults.yaml` file. node-defaults is always applied, and can be easily overridden with a user supplied config.